packages/design-system/src/components/Drawer/DrawerManager.stories.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Title, Subtitle, Description } from '@storybook/addon-docs';
+
+import Drawer from './Drawer';
+import { DrawerManager, useDrawerManager } from './DrawerManager';
+import { Button } from '../Button';
+
+export default {
+  title: 'Components/DrawerManager',
+  component: DrawerManager,
+  // The Drawer was overlapping the docs page, so customizing the docs page to remove the examples
+  parameters: {
+    docs: {
+      page: () => (
+        <>
+          <Title />
+          <Subtitle />
+          <Description />
+        </>
+      ),
+    },
+  },
+};
+
+const drawerContent1 = {
+  heading: 'Drawer Number One',
+  children: (
+    <>
+      <strong>Yar!</strong>
+      <p>
+        Prow scuttle parrel provost Sail ho shrouds spirits boom mizzenmast yardarm. Pinnace
+        holystone mizzenmast quarter crow&apos;s nest nipperkin grog yardarm hempen halter furl.
+        Swab barque interloper chantey doubloon starboard grog black jack gangway rutters.
```

Comment:
Lol, is this nautical nonesense? Maritime lorem ipsum?

Reply:
Yeah that one is pirate ipsum 😆

Reply:
Sarah brought up it would be nice to have a healthcare centered ipsum.